web_src/js/features/repo-legacy.js

```diff
@@ -442,11 +442,17 @@ async function onEditContent(event) {
     comboMarkdownEditor = await initComboMarkdownEditor($editContentZone.find('.combo-markdown-editor'));
 
     const $dropzone = $editContentZone.find('.dropzone');
+
+    // is setupDropzone doing anything? We already have have initGlobalDropzone to control all dropzones
```

Comment:
No. This is for the `$editContentZone.html($('#issue-comment-editor-template').html());` above, L441